example_school_library_decorator/student.rb

```diff
@@ -1,6 +1,7 @@
 require './person.rb'
 
 class Student < Person
+  belongs_to :classroom
 
```

Comment:
- [ ] Make sure that when setting the classroom for a student it also adds it to the classrooms' students.